tests/unit/classes/view/turnitintooltwo_view_test.php

```diff
@@ -53,7 +53,7 @@ public function test_output_header() {
         $turnitintooltwoview->output_header($pageurl, $pagetitle, $pageheading, true);
 
         $this->assertStringContainsString($pageurl, (string)$PAGE->url);
-        $this->assertEquals($pagetitle, $PAGE->title);
+        $this->assertStringContainsString($pagetitle, $PAGE->title);
```

Comment:
I'm fairly sure `assertStringContainsString` is only available in newer versions of phpunit - since this is the master / single branch it is probably used on 3.9 sites as well

Maybe replace it with strpos ?

Reply:
You might be correct, but the line right above is where I stole the check from, so it should be safe.